src/textual/message_pump.py

```diff
@@ -563,14 +563,18 @@ def _get_dispatch_methods(
             decorated_handlers = cls.__dict__.get("_decorated_handlers")
             if decorated_handlers is not None:
                 handlers = decorated_handlers.get(type(message), [])
-                for method, selector in handlers:
-                    if selector is None:
+                _sentinel = object()
```

Comment:
Why do we need a sentinal? Why not `None`?

If we do need it, let's move it outside the loop.

Reply:
The attribute might be there but it may be set to `None`, in which case I figured we’d like to error.

should I move the sentinel out and add a custom error if the attribute doesn’t provide a valid widget?

Reply:
The decorator does this for `control`, at import time. We should do this for the other attributes as well. In that case it is probably safe to assume the presence of the attribute, and that it is a widget.